doc/src/build/install.md

```diff
@@ -12,7 +12,7 @@ version 1.59.0 or higher in order to build and install Sui on your machine.
 To develop in Sui, you will need the Sui binaries. After installing `cargo`, run:
 
 ```shell
-$ cargo install --git https://github.com/MystenLabs/sui.git
+$ cargo install --git https://github.com/MystenLabs/sui.git sui
 ```
 
 This will put three binaries in your `PATH` (ex. under `~/.cargo/bin`) that provide these command line interfaces (CLIs):
```

Comment:
If you add `sui` it installs a single binary, the one named `sui`.
In the next sentence, we say "This will put three binaries in your `PATH` (ex. under `~/.cargo/bin`) that provide these command line interfaces (CLIs):". This is no longer true.

@bmwill @patrickkuo should be able to help update the doc after modifications in #1541 and #1588 (resp)

Reply:
Currently in our repo we have `faucet` package, which contain the faucet bin, and `sui` package, which contains rest of the bins, so selecting `sui` here will actually install all but faucet's binaries.

We don't need the sui package selection previously because we only had one bin creating package, but now we have 2 so it's causing error in cargo install... We might want to refactor this?

Reply:
> Currently in our repo we have `faucet` package, which contain the faucet bin, and `sui` package, which contains rest of the bins, so selecting `sui` here will actually install all but faucet's binaries.
> 
> We don't need the sui package selection previously because we only had one bin creating package, but now we have 2 so it's causing error in cargo install... We might want to refactor this?

Yes, please. I'm not sure how to proceed here. We should truly allow a fresh restart or update without issue.

Reply:
```
huitseeker@dapple.local➜~/tmp/sui(main)» ls sui/src/bin                                                                                                                                                                                             [20:04:15]
bench.rs       key_tool.rs    rest-server.rs rpc-server.rs  sui-move.rs    sui.rs         validator.rs   wallet.rs
huitseeker@dapple.local➜~/tmp/sui(main)» rg -A3 '\[\[bin\]\]'                                                                                                                                                                                       [20:04:19]
faucet/Cargo.toml
29:[[bin]]
30-name = "sui-faucet"
31-path = "src/main.rs"
```

I count 7 binary targets outside of the `faucet` crate. So :
- the command in the doc is correct, what comes after "this will put three binaries" isn't,
- where as we were mentioning three binaries being installed, we should now list and document up to 7,
- except some of those are probably not useful to most (e.g. validator, bench),
- so we should separate our binaries by use-case first, and then update the doc on how they work.

One easy first iteration would be to group the 3 binaries we were documenting already (sui, wallet, rest-server) in a separate crate, and the rest elsewhere.

Reply:
Opened https://github.com/MystenLabs/sui/issues/1639 to track